tests/framework/defs.py

```diff
@@ -26,7 +26,7 @@
 SECCOMP_JSON_DIR = FC_WORKSPACE_DIR / "resources/seccomp"
 
 # Maximum accepted duration of an API call, in milliseconds
-MAX_API_CALL_DURATION_MS = 300
+MAX_API_CALL_DURATION_MS = 1200
```

Comment:
Is there a way to increase this limit for that specific test only? This limit is actually pretty important.

Reply:
I don't think that we should increase the API call duration this long for all the framework. We should check if we can do it temporarily for a test as @georgepisaltu suggested.

Reply:
agree, one thing which is definitely supported by our framework is to disable API call measurements for specific tests or calls (see snapshot creation api calls).